go.mod

```diff
@@ -39,7 +39,7 @@ replace (
 )
 
 require (
-	github.com/Azure/azure-sdk-for-go v43.0.0+incompatible
+	github.com/Azure/azure-sdk-for-go v48.0.0+incompatible
```

Comment:
It looks like this was available back in 36.2, is the upgrade to 48.0 necessary ? If not we should revert to 43 being this close to a release.

Reply:
It may not be necessary. I'll check.

Reply:
As far as the `containerservice` module goes, the current version in this file does _not_ have `LoadBalancerSku` available. However, I am unsure how the path `github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2018-03-31/containerservice` relates to `github.com/Azure/azure-sdk-for-go <semver>`. Regardless, the current module does not have the functionality we need.

https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2018-03-31/containerservice

Reply:
Not sure about that link, but its available in the 36.2 codebase: https://github.com/Azure/azure-sdk-for-go/blob/v36.2.0/profiles/preview/preview/containerservice/mgmt/containerservice/models.go#L46-L50

Reply:
Confirmed that the YYYY-MM-DD module is a wrapper around a SemVer module for the Azure SDK. While `v43` is technically used, the `2018-03-31` module behaves differently, exposing different parts of the underlying library. We need to update the Azure API as a result.

Reply:
Makes sense, although I think we should keep that date a bit lower then. We've seen instability with AKS SDK upgrading to quickly. Lets stick with their original `2019-10-01`